src/component/registry/types.ts

```diff
@@ -510,6 +510,7 @@ export type Stroke = StyleWithOpacity & {
    * - It can be used when the style is first updated and then needs to be reset to its initial value.
    * - It doesn't use the color set in the BPMN source when the "BPMN in Color" support is enabled. It uses the color defined in the BPMN element default style.
    */
+  // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
```

Comment:
**suggestion:** add a description to explain why the rule is disable.
This applies to the whole file